.circleci/config.yml

```
@@ -17,7 +17,7 @@ jobs:
       - run:
           name: pull changes from merge
           command: |
-            [[ ! -z $CI_PULL_REQUEST ]] && git pull --ff-only origin "refs/pull/${CI_PULL_REQUEST//*pull\//}/merge"
+            if [[ -v CI_PULL_REQUEST ]] ; then git pull --ff-only origin "refs/pull/${CI_PULL_REQUEST//*pull\//}/merge" ; fi
```

Maybe circleci doesn't like `-v` ? 
```suggestion
            if [[ "$CI_PULL_REQUEST" != "" ]] ; then git pull --ff-only origin "refs/pull/${CI_PULL_REQUEST//*pull\//}/merge" ; fi
```